src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/build.py

```diff
@@ -401,11 +401,13 @@ def _pack_source_code(source_location, tar_file_path, docker_file_path):
     def _ignore_check(tarinfo, parent_ignored, parent_matching_rule_index):
         # ignore common vcs dir or file
         if tarinfo.name in common_vcs_ignore_list:
-            logger.debug(".dockerignore: ignore vcs file '%s'", tarinfo.name)
+            logger.warning("Excluding '%s' based on default ignore rules", tarinfo.name)
```

Comment:
Is this logging going to be overwhelming?

Reply:
It only logs top folder and file name.